package.json

```diff
@@ -149,42 +205,37 @@
     "settings": {
       "react": {
         "version": "^16.3.0"
+      },
+      "import/resolver": {
+        "@typescript-eslint/parser": [
+          ".ts",
+          ".tsx"
+        ]
       }
     }
   },
   "prettier": {
     "semi": false,
     "singleQuote": true
   },
-  "lint-staged": {
```

Comment:
Was there a reason for removing `lint-staged` + `husky`?

Reply:
I think we could/should adapt this to lint ts files (and `--fix` what it can automatically).

Reply:
Yes, it's going back in once I'm done (it's in my todo in the OP). I've had issues with husky testing things while performing typescript ports in the past and it's best to just leave it off once the migration is done.